spec/features/results_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'shows winner' do
+  scenario 'check battle results' do
+    fill_in_name
+    choose "Rock"
+    click_button "Play"
+    expect(page).to have_content('You')
```

Comment:
This test would be better if it checked the full output was Rock - it might be worth testing the other potential outputs too.